plugins/pagerduty/src/index.ts

```diff
@@ -23,6 +23,7 @@ export {
   isPluginApplicableToEntity as isPagerDutyAvailable,
   PagerDutyCard,
 } from './components/PagerDutyCard';
+export { TriggerButton } from './components/TriggerButton';
```

Comment:
This line is what is causing the build error @grantila.

Because this exports a React component to the public package, you need to have a the `@types/react` dep as a `dependencies` entry.

But maybe you don't even need to export this as a component in the public package at all?

Reply:
Thanks!

Yeah one idea with this PR is to actually expose the TriggerButton separately.

Reply:
hmm ok so now the build is failing I think because we are re-exporting the react types and we already have them some place else. I'm not sure what the fix is here, @rugvip probably knows a fix, although the only thing I can think about is basically creating a duplicate type which doesn't use `PropsWithChildren` instead.

Reply:
I added a dep on @types/react 17 🙈 which is probably why. Let's see if it passes with 16.

Reply:
Yep that totally worked! Thought it was a little strange. There's just some test failures left :)